bpf/kmesh/probes/probe.h

```diff
@@ -61,10 +61,9 @@ static inline void observe_on_close(struct bpf_sock *sk)
 
     storage = bpf_sk_storage_get(&map_of_sock_storage, sk, 0, 0);
     if (!storage) {
-        BPF_LOG(ERR, PROBE, "close bpf_sk_storage_get failed\n");
-        return;
+        BPF_LOG(INFO, PROBE, "TCP Connection closed or map_of_sock_storage cleaned up\n");
```

Comment:
Are you able to give an actual example?
Because storage is created when the link is established, this is handled when the link is closed. If the link is closed before this processing, then the whole processing logic is problematic

Reply:
> Are you able to give an actual example? Because storage is created when the link is established, this is handled when the link is closed. If the link is closed before this processing, then the whole processing logic is problematic

The ‘bpf_sk_storage_get ’ function is a helper function for getting bpf-local-storage from a socket (sk). It is similar to getting a value from a map with sk as the key, but bpf-local-storage is actually stored locally on the sk, not on the map.
I read in [https://docs.kernel.org/bpf/map_sk_storage.html](https://docs.kernel.org/bpf/map_sk_storage.html) that the ‘BPF_MAP_TYPE_SK_STORAGE’ type of map is recycled when the socket is deleted, and in the case that occurs here it should mostly return NULL, and the logs will print the err logs repeatedly.